programs/fullbench.c

```diff
@@ -188,7 +188,11 @@ static int local_count8v2(void* dst, size_t dstSize, const void* src, size_t src
     memset(count, 0, sizeof(count));
 
     while (ptr != end) {
+#if defined (__cplusplus) && (__cplusplus >= 201703L)
+        U64 bs = next;
+#else
         register U64 bs = next;
```

Comment:
we might just as well remove the `register` here, 
I don't see how it brings anything significant.